### ISSUE TITLE
Redgifs issue with . in username

### ISSUE THREAD

**Issue by bloodaxis** [RipMeApp/ripme#1786]
* Ripme version: 1.7.93
* Java version: java version "1.8.0_231"
Java(TM) SE Runtime Environment (build 1.8.0_231-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.231-b11, mixed mode)
* Operating system: Microsoft Windows [Version 10.0.17763.914]
* Exact URL you were trying to rip when the problem occurred: https://www.redgifs.com/users/katie.anders
* Please include any additional information about how to reproduce the problem: Just try to rip that url.

## Expected Behavior

Expected behavior is to download the mp4s of the clips.

## Actual Behavior

As you can see in the attached picture/log it tries to download from https://www.redgifs.com/users/katie rather than the correct katie.anders username.

![javaw](https://user-images.githubusercontent.com/57848972/98024106-85523780-1e10-11eb-88d4-33182031d54b.jpg)
[ripme.log](https://github.com/RipMeApp/ripme/files/5483165/ripme.log)

**Comment by borderline232** [comment]
If you want to do this yourself, you can add a dot to the regex found here
https://github.com/RipMeApp/ripme/blob/e0d40df5adc6618d962d939edaba8ed620f1bd78/src/main/java/com/rarchives/ripme/ripper/rippers/RedgifsRipper.java#L57-L60
to
`^https?://[wm.]*redgifs\\.com/users/([a-zA-Z0-9_.-]+).*$` which will detect the dot in usernames

If you don't know how to do this, then I'll try and update my fork (https://github.com/borderline232/ripme) with this change, because the owner of this ripme takes a while to update sometimes and try it there.

**Comment by bloodaxis** [comment]
Yeah I've got no clue how anything works here so I'll wait for you to update yours. 👍

**Comment by borderline232** [comment]
@bloodaxis Alright, I've added those changes to the newest release on my fork here (https://github.com/borderline232/ripme/releases/tag/1.7.96) lemme know if you have any issues
